extensions/sql-migration/src/constants/strings.ts

```diff
@@ -230,6 +232,7 @@ export const RESOURCE_GROUP_CREATED = localize('sql.migration.rg.created', "Reso
 export const NAME_OF_NEW_RESOURCE_GROUP = localize('sql.migration.name.of.new.rg', "Name of new Resource group");
 // common strings
 export const LEARN_MORE = localize('sql.migration.learn.more', "Learn more");
+export const LEARN_MORE_ABOUT_PRE_REQS = localize('sql.migration.learn.more.pre.reqs', "Learn more about things you need before starting a migration.");
```

Comment:
NIT: did we run these texts with a content writer?
i.e. "Learn more about things" seems a bit casual?

Reply:
no good point, I improvised using the group heading.
![image](https://user-images.githubusercontent.com/61598682/127281168-60470555-f821-40ab-828a-691dd63e02f9.png)

Reply:
I confirmed the current text with the product owner is good.